### PR TITLE
drivers/spi/spi_bitbang: Fix build warnings

### PR DESCRIPTION
**Pull request by SPRESENSE**
## Summary
Fix build warnings of spi_bitbang.c

## Impact
none

## Testing
Build with CONFIG_SPI_BITBANG=y

### CODE REVIEW

**Review by jerpelea — APPROVED** [review]

File: drivers/spi/spi_bitbang.c [hunk 1]
```diff
@@ -192,7 +192,7 @@ static void spi_select(FAR struct spi_dev_s *dev, uint32_t devid,
 {
   FAR struct spi_bitbang_s *priv = (FAR struct spi_bitbang_s *)dev;
 
-  spiinfo("devid=%d selected=%d\n", devid, selected);
+  spiinfo("devid=%ld selected=%d\n", devid, selected);
```
Comment (pkarashchenko) [inline drivers/spi/spi_bitbang.c:195]:
```suggestion
  spiinfo("devid=%" PRIu32 " selected=%d\n", devid, selected);
```

File: drivers/spi/spi_bitbang.c [hunk 2]
```diff
@@ -220,7 +220,7 @@ static uint32_t spi_setfrequency(FAR struct spi_dev_s *dev,
 
   DEBUGASSERT(priv && priv->low->setfrequency);
   actual = priv->low->setfrequency(priv, frequency);
-  spiinfo("frequency=%d holdtime=%d actual=%d\n",
+  spiinfo("frequency=%ld holdtime=%ld actual=%ld\n",
```
Comment (pkarashchenko) [inline drivers/spi/spi_bitbang.c:223]:
```suggestion
  spiinfo("frequency=%" PRIu32 " holdtime=%" PRIu32 " actual=%" PRIu32 "\n",
```

**Review by pkarashchenko — CHANGES_REQUESTED** [review]
Requesting a change since solution is compiler dependent

**Comment by SPRESENSE** [comment]
@pkarashchenko Thanks for your change request. 
I fixed them.

**Review by btashton — APPROVED** [review]

**Review by pkarashchenko — APPROVED** [review]

**Pull request merged**